docs/blocks-scaffolding.md

```diff
@@ -0,0 +1,45 @@
+# Scaffolding blocks
+
+It is also possible to generate PHP, JS and CSS code for registering a Gutenberg block for a plugin or theme.
```

Comment:
> It is also possible

"also possible", in addition to what? How would you rephrase this opening sentence for greater clarity?

Reply:
One suggestion: start with _why_. Explain why it's useful to me to scaffold blocks, vs. some other option.

Reply:
Yes, it makes a lot of sense. WIll add some references to the existing boilerplate code repositories, too.